tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py

```diff
@@ -933,3 +933,69 @@ def process(self, raw, identifiers, frame_meta):
             results.append(DetectionPrediction(identifier, labels, scores, x_mins, y_mins, x_maxs, y_maxs))
 
         return results
+
+
+class SSDVGG16Adapter(Adapter):
+    __provider__ = 'ssd_vgg16'
```

Comment:
BTW, we have additional file for SSD adapters, please move this adapter to ssd.py

Reply:
Done